### PR TITLE
Feature/splash screen

### PR DESCRIPTION
**Pull request by agustingodoyc**
STORY
Coordinator and Splash Screen
DESCRIPTION
Add Coordinator implementation and Splash Screen
EVIDENCE
Oct-14-2022 18-10-13

### CODE REVIEW

File: TheMovieDB/.swiftlint.yml [hunk 1]
```diff
@@ -15,10 +15,10 @@ excluded:
...
   - Project/R.generated.swift
 
-force_cast: error
-force_try: error
-force_unwrapping: error
-implicitly_unwrapped_optional: error
+force_cast: warning
+force_try: warning
+force_unwrapping: warning
+implicitly_unwrapped_optional: warning
```
Comment (ezequielvelezglobant) [inline TheMovieDB/.swiftlint.yml]: Why did you change that, it must be errors.

File: TheMovieDB/TheMovieDB/Presentation/Coordinator/Coordinator.swift [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+import Foundation
+import UIKit
+
+protocol Coordinator {
+    var childCoordinator: [Coordinator] { get set }
+    var navigationController: UINavigationController { get set }
+    
+    func start()
+}
```
Comment (ezequielvelezglobant) [inline TheMovieDB/TheMovieDB/Presentation/Coordinator/Coordinator.swift]: should be inside a Protocol folder

File: TheMovieDB/TheMovieDB/Presentation/Coordinator/Storyboeared.swift [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+//
+//  Created by Agustin Godoy Cosser on 12/10/2022.
+//
+
+import Foundation
+import UIKit
+
+protocol Storyboarded {
+    static func instantiate() -> Self?
+}
```
Comment (ezequielvelezglobant) [inline TheMovieDB/TheMovieDB/Presentation/Coordinator/Storyboeared.swift]: should be inside a protocol folder

File: TheMovieDB/TheMovieDB/Presentation/View/SplashScreen/SplashScreenViewController.swift [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    
+    // MARK: - Properties
+    
+    weak var coordinator: MainCoordinator?
+    
+    // MARK: - View Life Cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
```
Comment (ezequielvelezglobant) [inline TheMovieDB/TheMovieDB/Presentation/View/SplashScreen/SplashScreenViewController.swift]: remove unnecessary comment

Comment (agustingodoyc) [reply TheMovieDB/TheMovieDB/Presentation/View/SplashScreen/SplashScreenViewController.swift]: [Splash Screen Fixed](https://github.com/agustingodoyc/TheMovieDB/pull/2/commits/47fb8e3c3fcc65019a7ede66e0e23d51c5dd707b)

File: TheMovieDB/TheMovieDB/Presentation/Coordinator/Storyboeared.swift (hunk 3 above)
Comment (agustingodoyc) [reply TheMovieDB/TheMovieDB/Presentation/Coordinator/Storyboeared.swift]: https://github.com/agustingodoyc/TheMovieDB/pull/2/commits/306178015c7980b8d5a5be341797b3c21ffb3e8d

File: TheMovieDB/TheMovieDB/Presentation/Coordinator/Coordinator.swift (hunk 2 above)
Comment (agustingodoyc) [reply TheMovieDB/TheMovieDB/Presentation/Coordinator/Coordinator.swift]: https://github.com/agustingodoyc/TheMovieDB/pull/2/commits/306178015c7980b8d5a5be341797b3c21ffb3e8d

File: TheMovieDB/.swiftlint.yml (hunk 1 above)
Comment (agustingodoyc) [reply TheMovieDB/.swiftlint.yml]: https://github.com/agustingodoyc/TheMovieDB/pull/2/commits/306178015c7980b8d5a5be341797b3c21ffb3e8d

File: TheMovieDB/.swiftlint.yml [hunk 5]
```diff
@@ -16,7 +16,7 @@ excluded:
   - Project/R.generated.swift
 
 force_cast: error
-force_try: error
+force_try: errr
```
Comment (ezequielvelezglobant) [inline TheMovieDB/.swiftlint.yml]: there is a missing "o"

**Review by ezequielvelezglobant — CHANGES_REQUESTED** [review]

File: TheMovieDB/.swiftlint.yml (hunk 5 above)
Comment (agustingodoyc) [reply TheMovieDB/.swiftlint.yml]: [Splash Screen Fixed](https://github.com/agustingodoyc/TheMovieDB/pull/2/commits/47fb8e3c3fcc65019a7ede66e0e23d51c5dd707b)

**Review by ezequielvelezglobant — APPROVED** [review]

**Review by AngelaLee18 — APPROVED** [review]
